### PR TITLE
Enable CI

### LINKED ISSUE #11: Enable CI on repo
Once #8 is implemented, CI should be enabled on the repo to prevent regressions

### PR DESCRIPTION
**Pull request by MichaelSimons**
fixes #11

### CODE REVIEW

**Comment by MichaelSimons** [comment]
@dotnet-bot test ci please

**Comment by MichaelSimons** [comment]
@dotnet-bot test ci please

**Comment by MichaelSimons** [comment]
@dotnet-bot test Debian please
@dotnet-bot test NanoServer please

**Comment by MichaelSimons** [comment]
@dotnet-bot test please

**Comment by MichaelSimons** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+platformList.each { platform ->
+    def newJobName = Utilities.getFullJobName(project, platform, isPR)
+
+    def newJob = job(newJobName) {
+        steps {
+            if (platform == 'NanoServer') {
+                batchFile("powershell -NoProfile -Command .\\Microsoft.DotNet.ImageBuilder\\build.ps1 -CleanupDocker")
+            }
+            else {
+                shell("docker build --rm -t runner -f ./Microsoft.DotNet.ImageBuilder/Dockerfile.linux.runner .")
```
Comment (ravimeda) [inline netci.groovy]: I think `--rm` can be removed.

File: netci.groovy [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+def isPR = true
+def platformList = ['Debian', 'NanoServer']
+
+platformList.each { platform ->
+    def newJobName = Utilities.getFullJobName(project, platform, isPR)
+
+    def newJob = job(newJobName) {
+        steps {
+            if (platform == 'NanoServer') {
+                batchFile("powershell -NoProfile -Command .\\Microsoft.DotNet.ImageBuilder\\build.ps1 -CleanupDocker")
```
Comment (ravimeda) [inline netci.groovy]: Though consistent with the other dotnet Docker repos, I wanted to clarify if `-Command` should be replaced with `-File`.

File: netci.groovy [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                shell("docker run -v /var/run/docker.sock:/var/run/docker.sock runner pwsh -File ./Microsoft.DotNet.ImageBuilder/build.ps1 -CleanupDocker")
+            }
+        }
+    }
+
+    if (platform == 'NanoServer') {
+        newJob.with {label('windows.10.amd64.serverrs3.open')}
+    }
+    else {
+        Utilities.setMachineAffinity(newJob, 'Ubuntu16.04', 'latest-or-auto-docker')
```
Comment (ravimeda) [inline netci.groovy]: I like this usage `'Ubuntu16.04', 'latest-or-auto-docker'` as opposed to defining a variable for each as done in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/4

File: netci.groovy (hunk 1 above)
Comment (MichaelSimons) [reply netci.groovy]: Removed - That is what I get for coping 😄

File: netci.groovy (hunk 2 above)
Comment (MichaelSimons) [reply netci.groovy]: I'm not sure there is a difference in this particular usage.  I think `File` is more appropriate.  Changed to File.

**Comment by MichaelSimons** [comment]
@dotnet-bot test ci please

**Pull request merged**